docs/pages/management/security/general-recommendations.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: General Security Recommendations
+description: Provides a simplified checklist of security measures to consider to protect enrolled resources and Teleport clusters.
+---
+
+If you host the Teleport cluster on your own network, review the following checklist for actions you can take to make 
+your infrastructure and Teleport cluster more secure and less vulnerable to different attack vectors. You can alleviate
+many of these security concerns by using Teleport Team or Teleport Cloud Enterprise.
+
+- Use specific role combinations to assign users the least privilege.
```

Comment:
I think this could use a mention of the alternative to specific role combinations to clarify what the user would be avoiding here.

Does this mean the same as, "Define roles with the narrowest permissions possible and assign a user to multiple roles to expand the resources they can access, rather than granting a smaller number of broader roles"?